Desktop/exellenteam_python/cake.py

```diff
@@ -0,0 +1,12 @@
+
+
+def get_recipe_price(prices,optional=[],**ingredients):
```

Comment:
The comments here are the same as in the other exercises - please document your function, and check if you can simplify your loop.